lib/flatpak-installation.c

```diff
@@ -174,6 +174,57 @@ flatpak_get_supported_arches (void)
   return (const char * const *)flatpak_get_arches ();
 }
 
+/**
+ * flatpak_get_system_installations:
+ * @cancellable: (nullable): a #GCancellable
+ * @error: return location for a #GError
+ *
+ * Lists the system installations according to the current configuration and current
+ * availability (e.g. doesn't return a configured installation if not reachable).
+ *
+ * Returns: (transfer full) (element-type FlatpakInstallation): an GPtrArray of
+ *   #FlatpakInstallation instances
```

Comment:
That was a mistake, I think the right thing to do is actually not to set the free func and return a fully transfer full GPtrArray. This would be consistent with what the other functions not returning a list do.